src/audit-whitelist.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
If the name of the file does not matter I think we should start thinking about replacing the use of `whitelist` with `allowedlist`

Reply:
Good shout. I would suggest potentially `allowlist` rather than `allowedlist` to be consist with what I have seen elsewhere?

Reply:
Used "whitelist" as that was the option name but i prefer "allowlist" 👍